src/Driver/PgSQL/Statement.php

```diff
@@ -81,8 +81,15 @@ public function bindValue($param, $value, $type = ParameterType::STRING): bool
             throw UnknownParameter::new((string) $param);
         }
 
-        $this->parameters[$this->parameterMap[$param]]     = $value;
-        $this->parameterTypes[$this->parameterMap[$param]] = $type;
+        if ($type === ParameterType::BOOLEAN) {
+            // convert to boolean as value got transformed to int earlier
+            $booleanValue                                      = (bool) $value;
+            $this->parameters[$this->parameterMap[$param]]     = ($booleanValue === false ? 'f' : 't');
```

Comment:
This is too verbose. It could be something like `... = (bool) $value ? 't' : 'f'`. Please remove redundant parenthesis.